### PR TITLE
ARTS-621 - add permission assign-role

### PR DESCRIPTION
**Pull request by shajidissa**
## Description
Please replace this text with a summary of the change and which issue is fixed.

### Changes
- [ARTS-621] - add permission assign-role

### Checklist:

- [x] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

[ARTS-621]: https://ndph-arts.atlassian.net/browse/ARTS-621

### CODE REVIEW

File: trials/ci-trial/services/init-service-dev.yml [hunk 1]
```diff
@@ -61,10 +61,12 @@ trial:
       permissions:
         - id: create-person
         - id: view-person
+        - id: assign-role
     - id: admin
       permissions:
         - id: create-person
         - id: view-person
+        - id: assign-role
```
Comment (sameerapurini) [inline trials/ci-trial/services/init-service-dev.yml]: Should the admin also be given 'assign-role' permissions @shajidissa

**Review by darrengsmithson — CHANGES_REQUESTED** [review]
Why are we adding assign-role to superuser AND admin? Wouldnt it be better to have different permissions for different roles? As it stands we have bootstrap and automation both having the same roles/permissions with no user having only create/view person?

**Comment by shajidissa** [comment]
Theres a county-admin role with create-person. From a requirements perspective admin and superuser should have assign-role, one for the BA's wdyt ?

**Review by sameerapurini — DISMISSED** [review]

**Comment by sameerapurini** [comment]
> Why are we adding assign-role to superuser AND admin? Wouldnt it be better to have different permissions for different roles? As it stands we have bootstrap and automation both having the same roles/permissions with no user having only create/view person?

Yes adding only to "super user" would help

**Review by darrengsmithson — APPROVED** [review]

**Review by sameerapurini — APPROVED** [review]

**Pull request merged**